src/ui/ui_editor_view.rs

```diff
@@ -0,0 +1,23 @@
+use ratatui::{layout::Rect, style::{Color, Style, Stylize}, text::{Line, Span}, widgets::Paragraph, Frame};
+
+use crate::app::App;
+
+/// Gère le rendu graphique de l'éditeur de texte
+pub fn ui_editor_view(app: &mut App, frame: &mut Frame, area: Rect) {
+    let text = std::str::from_utf8(&app.editor.text_buffer).unwrap();
+    let lines: Vec<&str> = text.lines().collect();
+    let mut new_lines = Vec::new();
+    for (i, line) in lines.iter().enumerate() {
+        new_lines.push(Line::from(vec![
+            Span::styled(format!("  {:<4} ", i + 1), Style::default().fg(Color::Rgb(0x7e, 0x82, 0x94))),
+            Span::styled(format!("{} ", line.to_string()), Style::default().fg(Color::White)),
```

Comment:
warning: `to_string` applied to a type that implements `Display` in `format!` args
  --> src/ui/ui_editor_view.rs:23:36
   |
23 |                 format!("{} ", line.to_string()),
   |                                    ^^^^^^^^^^^^ help: remove this
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#to_string_in_format_args
   = note: `#[warn(clippy::to_string_in_format_args)]` on by default
```suggestion
            Span::styled(line.to_string(), Style::default().fg(Color::White)),
```